### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by kannuto** [FuelLabs/forc-wallet#113] [Q&A]
please provide the address of the wallet you are going to sign this transaction with:fuel1vacuc6rplh5j0j5pcthy082ma7sjfr0z07kw6hdh040gu4g5yh5sh5dexs
Error: Provider error: Response errors; not enough resources to fit the target.
I need help with this, supposedly it could be a service error, so it waits several hours and keeps getting the same error, what do I do?

**Comment by kayagokalp** [comment]
Hi, 

It looks like you do not have enough funds to pay for your transaction. Is that so? Are you trying to deploy to a beta network?

**Reply by kannuto** [reply]
if I'm testing I'm going to cry

**Reply by kayagokalp** [reply]
If you can provide me some information like the beta chain you are trying to deploy, I can help you much better. Without any information, we cannot proceed to resolve your problem.
